test/infra/authenticated-api/authenticated-api.test.ts

```diff
@@ -99,6 +99,15 @@ describe("AuthenticatedApi", () => {
         {
           Name: "test-MyTestAuthenticatedApi",
           ProtocolType: "HTTP",
+          CorsConfiguration: {
+            AllowCredentials: true,
```

Comment:
comment: confirms that by default `AllowCredentials` is set to `true` to ensure backwards compatibility.